lang/src/org/partiql/lang/eval/ExprValueExtensions.kt

```diff
@@ -79,6 +81,9 @@ fun ExprValue.booleanValue(): Boolean =
 fun ExprValue.numberValue(): Number =
     scalar.numberValue() ?: errNoContext("Expected number: $ionValue", internal = false)
 
+fun ExprValue.dateValue(): LocalDate =
+    scalar.dateValue() ?: errNoContext("Expected timestamp: $ionValue", internal = false)
+
```

Comment:
Should be "date", not "timestamp" in this string, right?

Reply:
Oops. Thanks. Changed.